model/src/main/java/de/evoila/cf/broker/model/context/Context.java

```diff
@@ -0,0 +1,174 @@
+package de.evoila.cf.broker.model.context;
+
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class Context {
+
+    public static final String PLATFORM_CLOUDFOUNDRY = "cloudfoundry";
+
+    public static final String PLATFORM_KUBERNETES = "kubernetes";
+
+    private static Logger log = LoggerFactory.getLogger(Context.class);
+
+    @JsonProperty("platform")
+    private String platform;
+
+    @JsonProperty("organization_guid")
+    private String organizationGuid;
+
+    @JsonProperty("organization_name")
+    private String organizationName;
+
+    @JsonProperty("space_guid")
+    private String spaceGuid;
+
+    @JsonProperty("space_name")
+    private String spaceName;
+
+    @JsonProperty("instance_name")
+    private String instanceName;
+
+    @JsonProperty("clusterid")
+    private String clusterId;
+
+    @JsonProperty("namespace")
+    private String namespace;
+
+    private Map<String, Object> additionalFields;
+
+
+    public Context() {
+    }
+
+    public String getPlatform() {
+        return platform;
+    }
+
+    public void setPlatform(String platform) {
+        switch (platform) {
+            case PLATFORM_CLOUDFOUNDRY:
+            case PLATFORM_KUBERNETES:
+                break;
+            default:
+                throw new IllegalArgumentException("Only Cloudfoundry and Kubernetes are supported Platforms");
```

Comment:
Unchecked exception throw and no note about it anywhere. Who catches this?

Reply:
By Jackson